input/data/2005/2005A%26A...439.1013A/tev-000061.yaml

```diff
@@ -1,4 +1,25 @@
 source_id: 61
 reference_id: 2005A&A...439.1013A
 
-# TODO: add data
+data:
+  livetime: 58.6
+  excess: 2469
+  significance: 21
+
+pos:
+  ra: {val: 13h3m0.4s, err: 4.4s, err_sys: 20s}
+  dec: {val: -63d11m55s, err: 31s, err_sys: 20s}
+
+morph:
+  type: gauss
+
+  sigma: {val: 0.16, err: 0.02}
```

Comment:
Suggest to put `0.16d` here to be explicit about degrees.
I think you had a file earlier where you gave a `sigma` without units, and it was in `arcmin`.
I don't even know what our default is, best to be explicit in the input data files.